### PR TITLE
Feature/pass polltimeout as duration

### PR DESCRIPTION
**Pull request by iSchluff**
A duration is a bit easier to work with as the user doesn't have to worry about the precision required by the call.

Also tiny readme fix

### CODE REVIEW

File: srtgo.go [hunk 1]
```diff
@@ -344,16 +345,16 @@ func (s SrtSocket) PacketSize() int {
...
+	return time.Duration(s.pollTimeout) * time.Millisecond
 }
 
-// SetPollTimeout - Sets polling max time, in milliseconds, for connect/read/write operations.
+// SetPollTimeout - Sets polling max time, for connect/read/write operations.
 // Only applied when socket is in non-blocking mode.
-func (s *SrtSocket) SetPollTimeout(pollTimeout int64) {
-	s.pollTimeout = pollTimeout
+func (s *SrtSocket) SetPollTimeout(pollTimeout time.Duration) {
+	s.pollTimeout = int64(pollTimeout / time.Millisecond)
```
Comment (jeoliva) [inline srtgo.go:357]: what if we do the following?


```suggestion
	s.pollTimeout = pollTimeout.Millisecond()
```

File: srtgo.go [hunk 2]
```diff
@@ -344,16 +345,16 @@ func (s SrtSocket) PacketSize() int {
 	return s.pktSize
 }
 
-// PollTimeout - Return polling max time, in milliseconds, for connect/read/write operations.
+// PollTimeout - Return polling max time, for connect/read/write operations.
 // Only applied when socket is in non-blocking mode.
-func (s SrtSocket) PollTimeout() int64 {
-	return s.pollTimeout
+func (s SrtSocket) PollTimeout() time.Duration {
+	return time.Duration(s.pollTimeout) * time.Millisecond
```
Comment (jeoliva) [inline srtgo.go:351]: Even though result is exactly the same, for coherence, what do you think if we do this way?

```suggestion
	return time.Duration(s.pollTimeout * time.Millisecond)
```

File: srtgo.go (hunk 1 above)
Comment (iSchluff) [reply srtgo.go:357]: ah I didn't know that was an option. Yeah that's even better.

File: srtgo.go (hunk 2 above)
Comment (iSchluff) [reply srtgo.go:351]: that wouldn't work because we need to cast pollTimeout to a duration first.

**Comment by iSchluff** [comment]
Changed to include Recommendation 1

**Pull request merged**